src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.wasm.cs

```diff
@@ -49,7 +49,9 @@ private void OnForegroundChanged(DependencyPropertyChangedEventArgs e)
 		}
 
 		public TextBoxView(TextBox textBox, bool isMultiline)
-			: base(isMultiline ? "textarea" : "input")
+			// We need to use textarea regardless of isMultiline
+			// because "input" native HTML element can't have its text top-aligned.
+			: base("textarea")
```

Comment:
This somehow breaks #16982 😕 

@MartinZikmund Do you see why?

Reply:
That's an existing bug with master. If this XAML is used and long text is entered, then scrolling doesn't work.

```xml
<TextBox AcceptsReturn="true" Width="50" />
```

Reply:
This is natively reproducible with:

```html
<!DOCTYPE html>
<html>
<body>

<textarea style="width: 50px; white-space: pre; overflow:hidden;">This is some very long text to test textarea native scrolling behavior.</textarea>
</body>
</html>

```